ugvc/cnv/convert_cnv_results_to_vcf.py

```diff
@@ -0,0 +1,156 @@
+import pandas as pd
+#import numpy as np
+import os
+from os.path import join as pjoin
+#from matplotlib import pyplot as plt
+#from matplotlib.patches import Rectangle
+import argparse
+import logging
+import sys
+#import seaborn as sns
+import pysam
+from ugvc import logger
+import warnings
+warnings.filterwarnings('ignore')
+
+
+def run(argv):
+    """
+    converts CNV calls in bed format to vcf. 
+    input arguments:
+    --cnv_annotated_bed_file: nput bed file holding CNV calls.
+    --ref_genome_file: tab delimeted file holding reference genome chr ids with their lengths.
+    --out_directory: output directory
+    --sample_name: sample name    
+    output files:
+    vcf file: <sample_name>.cnv.vcf
+        shows called CNVs in vcf format. 
+    """
+    parser = argparse.ArgumentParser(
+        prog="cnv_results_to_vcf.py", description="converts CNV calls in bed format to vcf."
+    )
+    
+    parser.add_argument("--cnv_annotated_bed_file", help="input bed file holding CNV calls", required=True, type=str)
+    parser.add_argument("--ref_genome_file", help="tab delimeted file holding reference genome chr ids with their lengths", required=True, type=str)
+    parser.add_argument("--out_directory", help="output directory", required=False, type=str)
+    parser.add_argument("--sample_name", help="sample name", required=True, type=str)
+    parser.add_argument("--verbosity",help="Verbosity: ERROR, WARNING, INFO, DEBUG",required=False,default="INFO",)
+
+    args = parser.parse_args(argv[1:])
+    logger.setLevel(getattr(logging, args.verbosity))
+
+    header = pysam.VariantHeader()
+
+    # Add meta-information to the header
+    header.add_meta('fileformat', value='VCFv4.2')
+    header.add_meta('source', value='ULTIMA_CNV')
+
+    # Add sample names to the header
+    sample_name=args.sample_name
+    header.add_sample(sample_name)
+
+    header.add_line('##GENOOX_VCF_TYPE=ULTIMA_CNV')
+
+    # Add contigs info to the header
+    df_genome = pd.read_csv(args.ref_genome_file,sep='\t',header=None,usecols=[0,1])
```

Comment:
You only use pandas to read the file. Better to read using python's base function ,and then you won't need a dependency on Pandas

Reply:
keeping it as is for now.